src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-wide/Format-Wide.cs

```diff
@@ -50,7 +50,7 @@ public SwitchParameter AutoSize
             get
             {
                 if (_autosize.HasValue)
-                    return _autosize.Value;
+                    return _autosize;
```

Comment:
We can not assign nullable type to non-nullable one.
Below too.
```suggestion
                    return _autosize.GetValueOrDefault();
```

Reply:
You should do the same for rest of changes.